packages/@lwc/rollup-plugin/package.json

```diff
@@ -11,24 +11,25 @@
     "bugs": {
         "url": "https://github.com/salesforce/lwc/issues"
     },
-    "main": "src/index.js",
+    "main": "dist/commonjs/index.js",
     "license": "MIT",
     "scripts": {
-        "test": "jest"
+        "clean": "rm -rf dist/",
+        "build": "tsc"
     },
     "files": [
-        "src/*.js"
+        "dist/"
     ],
     "devDependencies": {
-        "@lwc/compiler": "1.7.12",
-        "@lwc/engine": "1.7.12"
+        "@lwc/compiler": "1.7.12"
     },
     "dependencies": {
         "@lwc/module-resolver": "1.7.12",
         "@rollup/pluginutils": "~4.0.0"
     },
     "peerDependencies": {
-        "rollup": "^1.2.0||^2.0.0"
+        "rollup": "^1.2.0||^2.0.0",
+        "lwc": "^1.7.12"
```

Comment:
why `lwc` is a peer dependency?

Reply:
This is the key part of this PR. I will explain this here and I also updated the PR description to add more context.

Currently, all the consumers of LWC have to install `@lwc/engine` (or now `@lwc/engine-dom`) for the LWC engine to be resolved. If they want synthetic shadow they also have to install `@lwc/synthetic-shadow`. If they want to run SSR they also have to install `@lwc/engine-server`. Finally, in order to build an application, they have to install `@lwc/rollup-plugin`.

This is a lot of packages to install and to keep up to date. Instead of managing multiple packages, this PR proposes to load all those modules from the `lwc` umbrella package. The module resolver can be used to [resolve those modules](https://github.com/salesforce/lwc/blob/pmdartus/server-env/packages/lwc/package.json#L27-L62) using the standard LWC module resolution. With this PR, you will only need to `lwc` greatly reducing the maintenance burden.